lib/install/config/webpack/shared.js

```diff
@@ -11,7 +11,16 @@ const ManifestPlugin = require('webpack-manifest-plugin')
 const extname = require('path-complete-extname')
 const { env, settings, output, loadersDir, resolvedModules } = require('./configuration.js')
 
-const extensionGlob = `**/*{${settings.extensions.join(',')}}`
+const numberOfExtensions = settings.extensions && settings.extensions.length
+
+if (numberOfExtensions === undefined || numberOfExtensions === 0) {
```

Comment:
just need: 

```js
if(!numberOfExtensions) // it will check for undefined and 0 anyway
```

Also, lets just call it `extensions` instead of `numberOfExtensions`

Reply:
I've pushed up a change to change the variable to be `const extensions = settings.extensions` and calling `length` in the two places that check its `length` (3a7c92cf3f6d707f3196462f032f5d97fda9000e).

I prefer this to using the name `extensions` to hold the `length` of `settings.extensions` as I think that could be a bit confusing. However, if you'd rather go with that, just let me know and I'll change it.